LobbyCompatibility/Models/LobbyDiff.cs

```diff
@@ -36,6 +36,12 @@ internal LobbyDiffResult GetModdedLobbyType()
             return LobbyDiffResult.Unknown;
         }
 
+        if (PluginDiffs.All(pluginDiff => pluginDiff.RequiredVersion == null))
```

Comment:
Is this actually correct logic?

Reply:
it works but there might be a better way to do it

ideally any lobby that has *no mods on the server side whatsoever* is marked as unknown, because they don't have LobbyCompatibility installed

Reply:
Why don't you check pluginDiff's pluginDiffResult for unknown instead? RequiredVersion is only null if the server is missing the mod, and in that case it's either a missing requirement, or a clientside mod that's compatible - but neither are "unknown"?

Reply:
I'll move this to a separate issue/PR, this is likely out of scope now